lgc/interface/lgc/Pipeline.h

```diff
@@ -241,9 +241,10 @@ struct ShaderOptions {
 struct ResourceNode {
   ResourceNode() {}
 
-  ResourceNodeType type;   // Type of this node
-  unsigned sizeInDwords;   // Size in dwords
-  unsigned offsetInDwords; // Offset in dwords
+  ResourceNodeType type;      // Type of this node
```

Comment:
The names `type` and `matchType` are a little confusing.  Could we rename `type` to something like `underlyingType` and then rename `matchType` to `resourceType`.  The problem  with a name like `matchType` is that the name says what you will do with it, but not what it is.  A variable name is best when it describes the data is contains, and the code can use it in appropriate ways.

Comment to describe the difference between the two will be useful as well.